package-lock.json

```diff
@@ -5359,14 +5359,14 @@
         },
         "wrappy": {
           "version": "1.0.2",
-          "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
+          "resolved": false,
           "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
           "dev": true,
           "optional": true
         },
         "yallist": {
           "version": "3.0.3",
-          "resolved": "https://registry.npmjs.org/yallist/-/yallist-3.0.3.tgz",
+          "resolved": false,
```

Comment:
Did you run `npm update` on this branch?

Reply:
I'm unsure if all these `resolved` fields should have switched to `false`, seems unexpected?

Reply:
I ran npm install.. but i thought that seemed a lil odd as well.

Reply:
I'm not sure if having all of these as `false` could cause issues, but it does seem unexpected and seems folks have fixed with [`rm -rf node_modules && npm install`](https://github.com/RocketChat/Rocket.Chat/issues/15408). Seems worth a try? cc: @DFurnes if you have any insights (maybe this is not a problem?)

Reply:
Yeah, it's frustrating how many little things like this seem to be wonky with npm. It can't hurt to try a lil' `rm -rf node_modules && npm install` and see if that sorts things out!

Reply:
Ok! i deleted node modules and reinstalled and I'm not seeing anything on my `git diff`

Reply:
@DFurnes & @katiecrane what do you think about leaving? or maybe i should rebase with master?

Reply:
Might be worth trying that again, but also deleting `package-lock.json` (to force it to regenerate from the `package.json` constraints). If that doesn't sort things out, though, I'm good with merging it in as-is. 👻

Reply:
ok, that seems to have updated even more of the packages lol 🙈

Reply:
That makes sense! For better of for worse, npm is really conservative with updating sub-dependencies when you have a lockfile (even when running `npm update`) so they can fall behind.

And it looks like this fixed the `resolved` weirdness, so that's good too! 😸

Reply:
Yeah I think it looks less funky now!